docs/conditional_rendering.md

```diff
@@ -0,0 +1,268 @@
+```python exec
+import reflex as rx
+
+from pcweb.base_state import State
+from pcweb.templates.docpage import docdemo_from
+from pcweb.pages.docs.component_lib.layout import render_cond
+```
+
+# Conditional Rendering
+
+We use the `cond` component to conditionally render components. The `cond` component acts in a similar way to a conditional (ternary) operator in python, acting in a similar fashion to an `if-else` statement.
+
+
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.alert_title("A link to the API reference cond docs will go here"),
+    status="info",
+)
+```
+```python eval
+rx.box(height="2em")
+```
+
+Here is a simple example to show how by checking the value of the state var `show` we can render either `blue` text or `red` text. 
+
+The first argument to the `cond` component is the condition we are checking. Here the condition is the value of the state var boolean `show`. 
+
+If `show` is `True` then the 2nd argument to the `cond` component is rendered, in this case that is `rx.text("Text 1", color="blue")`. 
+
+If `show` is `False` then the 3rd argument to the `cond` component is rendered, in this case that is `rx.text("Text 2", color="red")`.
+
+```python exec
+class CondSimpleState(rx.State):
+    show: bool = True
+
+    def change(self):
+        self.show = not (self.show)
+
+
+def cond_simple_example():
+    return rx.vstack(
+        rx.button("Toggle", on_click=CondSimpleState.change),
+        rx.cond(
+            CondSimpleState.show,
+            rx.text("Text 1", color="blue"),
+            rx.text("Text 2", color="red"),
+        ),
+    )
+```
+
+```python eval
+docdemo_from(CondSimpleState, component=cond_simple_example)
+```
+
+
+## Negation
+
+You can use the logical operator `~` to negate a condition. In this example we show that by negating the condition `~CondNegativeState.show` within the cond, we then render the `rx.text("Text 1", color="blue")` component when the state var `show` is negative.
```

Comment:
Instead of negation this section should have a general title for all var operation and link to the var operation docs as reference